NGitLab.Mock/Repository.cs

```diff
@@ -12,6 +12,7 @@ namespace NGitLab.Mock
 {
     public sealed class Repository : GitLabObject, IDisposable
     {
+        public const string DefaultBranch = "main";
```

Comment:
We should define the fallback default branch only once in the codebase. "main" is also specified here:

https://github.com/ubisoft/NGitLab/blob/cf0a15949dfc582607c0d58038f89bb69eb3c4a9/NGitLab.Mock/Project.cs#L41

Maybe replace that by
```cs
public sealed class Project : GitLabObject
{
    public string DefaultBranch { get; set; } = Repository.DefaultBranch;
}
```
?